packages/extension-core/src/config/posthog.ts

```diff
@@ -27,7 +27,7 @@ const unsafeProperties = [
 const talismanProperties = {
   appVersion: process.env.VERSION,
   appBuild: process.env.BUILD,
-  testBuild: DEBUG || ["qa", "ci"].includes(process.env.BUILD as string),
+  testBuild: DEBUG || ["dev", "qa", "ci"].includes(process.env.BUILD as string),
```

Comment:
I added it because I'd expect that if for some reason someone did set an auth token in dev, we'd want those events labelled as test events not prod events

Reply:
But FWIW, `DEBUG` is true unless `process.env.BUILD` is either `production` or `canary`

Reply:
ok, seems like it won't cause any harm